src/NzbDrone.Core/Notifications/Telegram/TelegramSettings.cs

```diff
@@ -32,6 +40,12 @@ public class TelegramSettings : IProviderConfig
         [FieldDefinition(3, Label = "NotificationsTelegramSettingsSendSilently", Type = FieldType.Checkbox, HelpText = "NotificationsTelegramSettingsSendSilentlyHelpText")]
         public bool SendSilently { get; set; }
 
+        [DefaultValue(true)]
+        [FieldDefinition(4, Label = "NotificationsTelegramSettingsSendMetadataLink", Type = FieldType.Checkbox, HelpText = "NotificationsTelegramSettingsSendMetadataLink")]
```

Comment:
For the Label and Helptext to correctly translate, please add the key and value to the `src/NzbDrone.Core/Localization/Core/en.json` file

Reply:
No need for this setting, it can all be done with the enum.